pkg/config/validation/validation.go

```diff
@@ -1083,32 +1050,32 @@ func ValidateParentAndDrain(drainTime, parentShutdown *types.Duration) (errs err
 		return
 	}
 
-	drainDuration, _ := types.DurationFromProto(drainTime)
-	parentShutdownDuration, _ := types.DurationFromProto(parentShutdown)
+	drainDuration := drainTime.GetSeconds()
+	parentShutdownDuration := parentShutdown.GetSeconds()
 
-	if drainDuration%time.Second != 0 {
+	if drainDuration < 1.0 {
```

Comment:
we are losing this check. Its about a user providing `1100ms` - we can only support second increments

Reply:
makes sense. I'll add a comment in this block as well - this is not really all that obvious.